packages/rn-tester/RNTesterPods.xcodeproj/project.pbxproj

```diff
@@ -586,7 +589,58 @@
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 			shellPath = /bin/sh;
-			shellScript = "export NODE_BINARY=node\nexport PROJECT_ROOT=$SRCROOT/..\nexport SOURCEMAP_FILE=../sourcemap.ios.map\n# export FORCE_BUNDLING=true\n\"$SRCROOT/../../scripts/react-native-xcode.sh\" $SRCROOT/../../packages/rn-tester/js/RNTesterApp.ios.js\n";
+			shellScript = "export NODE_BINARY=node\nexport SOURCEMAP_FILE=../sourcemap.ios.map\nexport PROJECT_ROOT=\"$SRCROOT/../../\"\n\"$SRCROOT/../../scripts/react-native-xcode.sh\" $SRCROOT/../../packages/rn-tester/js/RNTesterApp.ios.js\n";
```

Comment:
Looks like this line has still changed. Are these changes unintentional? Looks like it flipped the order of the definition of PROJECT_ROOT and SOURCEMAP_FILE, and no longer defines `FORCE_BUNDLING=true`. If these changes aren't necessary it would be great to remove them to make upgrading easier.

Reply:
These changes are for the RNTester project only. The only change was to change `PROJECT_ROOT` to fix it (with a different path). Previously, building in `release` mode was broken. Will make the diff more simple.